rasa/nlu/training_data/training_data.py

```diff
@@ -168,7 +168,10 @@ def examples_per_intent(self) -> Dict[Text, int]:
     @lazy_property
     def examples_per_response(self) -> Dict[Text, int]:
         """Calculates the number of examples per response."""
-        return dict(Counter(self.responses))
```

Comment:
Can we remove the `self.responses` completely now?

Reply:
No, `self.responses` is still used.